server/auth/authApi.ts

```diff
@@ -96,21 +97,23 @@ export const init = (app: any) => {
   app.post('/auth/local/resetPassword', async (req: any, res: any) => {
     try {
       const { email } = req.body
+      const i18n = await createI18nPromise('en')
```

Comment:
you cannot hardcode language.
server should respond with keys only. frontend will use i18n